packages/graphql/src/services/groups.ts

```diff
@@ -1591,7 +1596,9 @@ export async function publishGroupActivity(
   const updatedGroupActivity = await ctx.prisma.groupActivity.update({
     where: { id },
     data: {
-      status: GroupActivityStatus.PUBLISHED,
+      status: dayjs().isBefore(groupActivity.scheduledStartAt)
```

Comment:
Make sure that dayjs treats the time correctly also with respect to time zones, etc.  - otherwise use js dates